src/Orleans.Clustering.AzureStorage/Orleans.Clustering.AzureStorage.csproj

```diff
@@ -0,0 +1,37 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <PackageId>Microsoft.Orleans.Clustering.AzureStorage</PackageId>
+    <Title>Microsoft Orleans Membership Azure Table Storage</Title>
+    <Description>Membership provider for Azure Table Storage of Microsoft Orleans.</Description>
+    <PackageTags>$(PackageTags) Azure</PackageTags>
```

Comment:
might want to include both AzureTable and Table as tags

Reply:
AzureTable as a term is not a thing as of search: https://www.nuget.org/packages?q=Tags%3A%22AzureTable%22, Table is too common IMO.